.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+## Status
+- Done, In progress, etc
+
+## GitHub Issues addressed (`#IssueNumber`)
+- This PR closes 
+
+## What I did
+- 
+
+## Screenshots
+- Before
+- After
+
+## Tests
+- A brief explanation of tests done/written or how reviewers can test your work
+
+## Questions/Discussions/Notes
+- 
+
+## Merging the PR
+- Who should merge this PR?
+  - [ ] I will merge it myself
+  - [ ] The last reviewer to approve can merge it
+    - Should the commit history be preserved in the base branch, or is it okay to combine all of them into a single commit ("squash-merge")?
```

Comment:
Wonder if we should set some kind of criteria for this instead of letting the author decide
Ex. if more than half the PR is merge commits or typo fixes, pls squash

Reply:
If each commit in the PR is atomic and self-contained while being pretty big, merge commits can work. However, I think it's usually better to keep each PR small and squash-merge.